src/bin/utils/processors.ts

```diff
@@ -11,10 +11,9 @@ import type {
 import type { SessionToken } from '../../sessions/types';
 import path from 'path';
 import prompts from 'prompts';
-import * as grpc from '@grpc/grpc-js';
 import Logger from '@matrixai/logger';
 import Status from '../../status/Status';
-import * as clientUtils from '../../client/utils';
+import * as clientUtils from '../../client/utils/utils';
```

Comment:
This isn't actually what I meant. In many other situations what we do when we have this kind of code is to do this:

```
utils/index.ts
utils/x.ts
utils/y.ts
```

You don't want to do `utils/utils.ts`. That defeats the purpose of putting it into a directory. It's to have file-level separation which can be more efficient for importing or navigation and code review.

Reply:
Indeed there is another one with:

```
utils.ts
utils/x.ts
```

I don't like this as it is confusing, many other languages have confusing overlaps here, it's better to use `utils/index.ts`.

You can have broad root-level utilities in `utils/index.ts`, and you can specialise certain utilities into other files.

Reply:
Ok, so you're saying that the methods in `utils/utils.ts` should be moved into the `utils/index.ts`?

Reply:
Not necessarily, it's about splitting up the methods into files. I'm looking into the `rpc` now to see what factorisation can occur.

Reply:
I'm confused then. I created the utils directory in the client and `rpc` domain to store the `utils.ts` and `middleware.ts` files together. I can maybe split the `utils.ts` files up more? haven't I done what you are saying here?

Reply:
Will leave this to later, this requires analysis of the middleware functions and how best to organise them.